package.json

```diff
@@ -11,6 +11,7 @@
     "bootstrap-colorpicker": "2.5.2",
     "chart.js": "2.7.0",
     "compass-mixins": "0.12.10",
+    "enzyme-adapter-react-15": "^1.0.1",
```

Comment:
@Zapix Are these libraries needed for this PR?

Reply:
@jasonblais Yeah, I think so. This is as per enzyme recommendation.
@Zapix I suggests to fix the version like `enzyme-adapter-react-15": "1.0.1`

Reply:
@jasonblais @saturninoabril What should I do with current dependencies? Move to another PR?

Reply:
@saturninoabril I see you submitted a separate PR which adds these dependancies: https://github.com/mattermost/mattermost-webapp/pull/111

Should that PR be merged first, and then @Zapix can help rebase his branch afterwards?

Reply:
@Zapix Just update this PR with fix version like `enzyme-adapter-react-15": "1.0.1`. That is without `^`.
@jasonblais Whichever came first. The last one will do the rebase.

Reply:
I"ve updated pr please check